src/style.css

```diff
@@ -1 +1,86 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
```

Comment:
The `font-weight` for the body is set to 300, but it should be 400 according to the task requirements.